src/EventSubscriber/OgEventSubscriber.php

```diff
@@ -91,6 +91,12 @@ public function provideDefaultOgPermissions(PermissionEventInterface $event) {
         'description' => $this->t('Edit the group entity.'),
         'default roles' => [OgRoleInterface::ADMINISTRATOR],
       ]),
+      new GroupPermission([
+        'name' => OgAccess::DELETE_GROUP_PERMISSION,
+        'title' => $this->t('Delete group'),
+        'description' => $this->t('Delete the group.'),
```

Comment:
Either we add group here:

```suggestion
        'description' => $this->t('Delete the group entity.'),
```

Or we remove group from
`        'description' => $this->t('Edit the group entity.'),
`